### DISCUSSION TITLE
how to config cursor theme in qtile?

### DISCUSSION THREAD

**Discussion by superiums** [qtile/qtile#4506] [General]
GTK apps use the cursor as i set in gtk settings.
but in bar, or some other apps ( foot/ alacrritty ...), it use another cursor theme.

how to config it?

**Comment by Im-shamo** [comment]
I created a gist that should help setting a cursor theme. [Gist](https://gist.github.com/Im-shamo/64d7efba89db1a816c2c70e5ba01f770)
